extensions/tensorflow/TFConvertImageToTensor.cpp

```diff
@@ -325,7 +325,7 @@ int64_t TFConvertImageToTensor::ImageReadCallback::process(const std::shared_ptr
   auto num_read = stream->read(tensor_->flat<unsigned char>().data(),
                                    static_cast<int>(stream->size()));
 
-  if (num_read != stream->size()) {
+  if (static_cast<uint64_t>(num_read) != stream->size()) {
```

Comment:
same here: `stream->size()` returns `size_t`, so we should preferably cast `num_read` to that. Same at line 340.

Reply:
Thanks! Updated in 7f2abdc0ae7a9f55c07339edc9c9d84de409c4de